client/src/components/Header/Nav.jsx

```diff
@@ -0,0 +1,62 @@
+import "../../../node_modules/bootstrap/dist/css/bootstrap.min.css";
+
+const Nav = () => {
+  return (
+    <nav className="navbar navbar-expand-lg navbar-light bg-light">
```

Comment:
@Boris713 Did you check if you are allowed to bootstrap? or you are expected to write custom CSS throughout the project

Reply:
I can ask again but they had no issue when we asked to use it for weekly projects